content/03.results.md

```diff
@@ -224,11 +224,12 @@ A summary of all the consensus cell types observed in all other ScPCA samples ca
 ## Analysis of bulk RNA-seq
 
 Several projects in the ScPCA Portal contain bulk RNA-seq data in addition to single-cell/nuclei RNA-seq data.
-Previous research has suggested that, compared to bulk RNA-seq, single-cell/nuclei RNA-seq technologies may fail to capture certain cell types [@doi:10.1038/s41587-020-0465-8], for example due to technical aspects of library preparation.
+Previous research has suggested that, compared to bulk RNA-seq, single-cell/nuclei RNA-seq technologies may fail to capture certain cell types [@doi:10.1038/s41587-020-0465-8], for example, due to technical aspects of library preparation.
 We therefore asked whether we could identify differences in biological signal between these two modalities that may suggest distinct cell type distributions.
-For this analysis, we specifically focused on ScPCA projects with solid tumors, considered only non-multiplexed samples with both sequencing modalities, and excluded low-quality single-cell/nuclei libraries.
-In total, we analyzed 97 samples across five projects: `SCPCP00001`, `SCPCP000002`, `SCPCP000006`, `SCPCP000009`, and `SCPCP000017`.
-Projects `SCPCP000001` and `SCPCP000002` comprise high- and low-grade gliomas, respectively, and were sequenced at the bulk and single-cell levels, and `SCPCP000006`, `SCPCP000009`, and `SCPCP000017` comprise Wilms tumors, CNS tumors, and osteosarcomas, respectively, and were sequenced at the bulk and single-nuclei levels.
+We specifically focused on ScPCA projects with solid tumors, considered only non-multiplexed samples with both sequencing modalities, and excluded low-quality single-cell/nuclei libraries.
```

Comment:
I feel like excluding multiplexed is not so important as to be the first thing mentioned.
```suggestion
We specifically focused on ScPCA projects with solid tumors, considering only samples with both sequencing modalities, and excluded low-quality single-cell/nuclei libraries and multiplexed samples.
```